sources/packages/web/src/views/student/StudentAccountActivityFormModel.ts

```diff
@@ -0,0 +1,4 @@
+import { StudentRestriction } from "@/types";
+export interface StudentAccountActivityFormModel {
```

Comment:
Is it something that will be used anywhere else? If not, I believe that it can be defined inside the component.